materials/flanker_golf_tasks/readme.md

```diff
@@ -1,7 +1,7 @@
-# Task1
+# Flanker and golf tasks for putt-putt-dataset
 
 ### Instructions
-This folder contains the protocols, scripts, and stimuli used to create this experimental task. Replace "Task1" in the header above with a descriptive name for the task; rename the "task1" folder in like manner.
+This folder contains the protocols, scripts, and stimuli used to create this experimental task. 
```

Comment:
But this folder doesn't actually contain anything yet?  It should have the full PsychoPy experiment for the flanker (under scripts) and then maybe a detailed overview of the setup of the golf-putting task?  Feel free to collapse the levels and make the flow more intuitive within the "materials" folder as a whole.  The purpose of this folder is to enable replicability of your methods.

Reply:
Also, please separate the flanker and golf task materials, since they are independent. :)